pm_yaml_config/pm_yaml_config.xml

```diff
@@ -72,58 +72,58 @@
+            <param type="float" argument="geom_force_tol" value="0.05" label="Geometry optimization tolerance" help="Tolerance on geometry optimisation in units of eV/Å."/>
+            <param type="integer" argument="random_seed" label="Random Seed" optional="true" help="Random seed for pymuonsuite."/>
         </section>
         <section name="calculator_params" expanded="true" title="Calculator parameters">
             <conditional name="calculator_cond">
-                <param type="select" display="radio" label="Optimization calculator" argument="calculator" name="calculator" value="uep" help="Calculator to generate structure files for. Must be a single word or a comma seperated list of values. Currently supported calculators are CASTEP, DFTB+ and UEP. Can also select 'all' to generate files for all calculators.">
+                <param type="select" display="radio" label="Optimization calculator" argument="calculator" value="uep" help="Calculator to generate structure files for. Must be a single word or a comma seperated list of values. Currently supported calculators are CASTEP, DFTB+ and UEP. Can also select 'all' to generate files for all calculators.">
                     <option value="uep">UEP</option>
                     <option value="castep">CASTEP</option>
                     <option value="dftb+">DFTB+</option>
                     <option value="all">all</option>
                 </param>
                 <when value="uep">
-                    <param type="float" argument="uep_gw_factor" name="uep_gw_factor" optional="true" value="5.0" label="Gaussian width factor" help="Gaussian width factor for UEP calculation. Higher values will make the potential of atomic nuclei closer to the point-like approximation but may introduce artifacts."/>
-                    <param type="text" argument="uep_chden" name="uep_chden" optional="true" label="Path to .den_fmt and .castep files" help="Name of the .den_fmt file containing the electronic density for an Unperturbed Electrostatic Potential optimisation. The corresponding .castep file must be with the same seedname. Only required for use outside Muon Galaxy"/>
+                    <param type="float" argument="uep_gw_factor" optional="true" value="5.0" label="Gaussian width factor" help="Gaussian width factor for UEP calculation. Higher values will make the potential of atomic nuclei closer to the point-like approximation but may introduce artifacts."/>
+                    <param type="text" argument="uep_chden" optional="true" label="Path to .den_fmt and .castep files" help="Name of the .den_fmt file containing the electronic density for an Unperturbed Electrostatic Potential optimisation. The corresponding .castep file must be with the same seedname. Only required for use outside Muon Galaxy"/>
```

Comment:
The label is suboptimal here as the user never specifies a path. Not sure what the input here should be?

Reply:
Mh, if this really takes a path an does something with it - its actually quiet dangerous.

Reply:
@patrick-austin has it right - if the generated file is used within Galaxy, anything entered in this field gets replaced when the next tool is run (replaced by the path to the `data` type input). So it shouldn't be dangerous, just weird design...

Thanks for flagging the label confusion. We should definitely discuss whether it's necessary to keep this as part of the larger issue.